sklearn/metrics/pairwise.py

```diff
@@ -138,6 +140,22 @@ def check_pairwise_arrays(
         .. versionchanged:: 0.23
            Accepts `pd.NA` and converts it into `np.nan`.
 
+        .. deprecated:: 1.6
+           `force_all_finite` was renamed to `ensure_all_finite` and will be removed
+           in 1.8.
+
+    ensure_all_finite : bool or 'allow-nan', default=True
```

Comment:
I'm happy with the approach of using `None` as the default and documenting that it's `True`.